config.def.h

```diff
@@ -34,7 +34,7 @@
 - uid (uid of current user) [argument: NULL]
 - uptime (uptime) [argument: NULL]
 - username (username of current user) [argument: NULL]
-- vol_perc (alsa volume and mute status in percent) [argument: soundcard]
+- vol_perc (oss/alsa volume and mute status in percent) [argument: /dev/mixer]
```

Comment:
Maybe "mixer file, /dev/mixer" instead of "/dev/mixer" as comment? Would simplify it for newbies maybe.